src/util/hass-util.html

```diff
@@ -69,6 +71,16 @@
   }
 };
 
+// If root has lastChild, set 'isVisible' attribute of that child to false.
+window.hassUtil.resetDynamicContent = function (root) {
```

Comment:
When we remove the child from the DOM, the image is no longer attached to the DOM either and thus it should stop streaming? Otherwise that might be a browser bug.